src/components/CartItem/CartItem.tsx

```diff
@@ -30,47 +31,61 @@ const CartItem = ({
   onChangeDelete,
 }: CartItemProps) => {
   const [amountOfProduct, setAmountOfProduct] = useState<number>(quantity);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
 
   const handleQuantityChange = (newQuantity: number) => {
     setAmountOfProduct(newQuantity);
     onQuantityChange(id, newQuantity);
   };
 
   const handleClick = () => {
+    setIsLoading(true);
     onChangeDelete(id);
+
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 500);
```

Comment:
Why we need to setTimeout here

Reply:
I need it to simulate the api call process

Reply:
Please note comments if you mock or do something cause confused

Reply:
fixed: add comment code